xbmc/cores/DllLoader/exports/emu_msvcrt.cpp

```diff
@@ -1633,7 +1631,7 @@ extern "C"
     int ret;
 
     ret = dll_fgetpos64(stream, &tmpPos);
-#if !defined(TARGET_POSIX) || defined(TARGET_DARWIN) || defined(TARGET_FREEBSD) || defined(TARGET_ANDROID)
+#if !defined(__GLIBC__) || defined(TARGET_DARWIN) || defined(TARGET_FREEBSD) || defined(TARGET_ANDROID)
```

Comment:
ok, thats what I suspected. Lets use a the define as you suggested, cause I don't like tinkering with our TARGET_POSIX